x/leverage/keeper/liquidate.go

```diff
@@ -145,7 +155,14 @@ func ComputeLiquidation(
 
 	// We will track limiting factors by the ratio by which the max repayment would need to be reduced to comply
 	ratio := sdk.OneDec()
-	// Repaid value cannot exceed borrowed value times close factor
+	// Repaid value cannot exceed borrowed value times close factor.
+
+	//
+	//
+	// TODO 2: This needs adjustment against rounding
+	//
+	//
```

Comment:
is this going to be updated in this PR?

Reply:
yeah - it's the last thing to look at. unfortunately the obvious fix breaks the abstraction I had going where we don't call oracle functions inside liquidate, so I'm looking for a way around.

Reply:
Fixed now - I moved close factor (which relied on token price) one function up, and made it more precise by using `TokenValue` of total borrow instead of (base) `TokenPrice`